tools/install-unprivileged.sh

```diff
@@ -284,7 +337,12 @@ cd ..
 
 set -e
 
+# remove .build-id files seen on el8 or later
+rm -rf lib/.build-id
+rmdir lib 2>/dev/null || true
```

Comment:
Since this is my change, I think you should instead rebase on the current main and force-push.